std_msgs/Bool c1
std_msgs/Bool c2
std_msgs/Bool c3
